### PR TITLE
Mpeg:Only allow firmware >= 3 for warmup

### LINKED ISSUE #13996: Street Supremacy [ULUS10069] - black screen on latest nightly
### What happens?
After the initial logos, it's stuck on a black screen. It's persistent on all backends.

### What *should* happen?
It should play video and go to menus as normal. Possible regression due to the game fully working on previous versions of PPSSPP.

### What hardware, operating system, and PPSSPP version?  On desktop and mobile phone, GPU matters for graphical issues.
NVIDIA Geforce 740m
Windows 10 x64
v1.10.3-1659-gbcd7856aa

### PR DESCRIPTION
**Pull request by sum2012**
Fix #13996

edit :and delete warmup in sceMpegGetAtracAu

### CODE REVIEW

File: Core/HLE/sceMpeg.cpp [hunk 1]
```diff
@@ -34,6 +34,7 @@
 #include "Core/Reporting.h"
 #include "GPU/GPUInterface.h"
 #include "GPU/GPUState.h"
+#include "CORE/HLE/sceKernelMemory.h"
```
Comment (hrydgard) [inline Core/HLE/sceMpeg.cpp:37]: This needs to be changed to "Core/HLE/sceKernelMemory.h" . Some systems are case sensitive.

**Comment by hrydgard** [comment]
Is this supported by testing on hw? I think this is a reasonably safe change anyway, just wondering.

**Comment by sum2012** [comment]
God eater don't give JPCSPTrace .
MegaMan power up firmware 2.6
Street Supremacy firmware 2.5
So that I give limit in firmware 3.0

**Comment by hrydgard** [comment]
OK, sounds reasonable.

**Comment by sum2012** [comment]
Add one more test :EA Replay firmware 2.71 :)

**Comment by unknownbrackets** [comment]
I think this is fine given how long we haven't had the warm up.  I remember testing and trying to simulate it before, and it was tricky.  I think it might actually have to do more with timing than number of calls, but I haven't tried testing again in a couple years now...

-[Unknown]

**Comment by sum2012** [comment]
I gave some test for more warm up test for other user for mpeg issue ,he said that my change is like  frame skip 1 effect.....

**Pull request merged**